objects/index.js

```diff
@@ -84,5 +84,8 @@ export const joinObjects = (objA, objB) => {
   isObject should also be set to true
 */
 export const addIsObjectProperty = (obj) => {
-  return obj['isObject'] = true; // I cant get this to work so I'm leaving it with what I thought should work
+  let newObj = {
+    isObject: true
+  };
+  return newObj;
```

Comment:
Going through this line by line:
Line 87->89: I am creating a new object with a property isObject with a value of true
Line 90: Return new object

You are creating and returning a new object, you aren't modifying the existing one

Reply:
yeah thats why i wrote the extra tests. thought i mightve fucked that up!

Reply:
Have another crack, im about with an eye on github so will be around